dsl/src/commonTest/kotlin/it/unibo/collektive/alignment/BranchAlignment.kt

```diff
@@ -1,8 +1,7 @@
 package it.unibo.collektive.alignment
```

Comment:
We should have a coherent naming for classes. This is `BranchAligment`, other classes end in `Test`. Let us pick one and stick to the decision